File: python/easy/771.py
class Solution:
    def numJewelsInStones(self, J, S):
        """
        :type J: str
        :type S: str
        :rtype: int
        """
        table = [False] * 70 
        number_of_j = 0
        for j in J:
            table[ord(j) - 65] = True 
        for s in S:
            if table[ord(s) - 65]:
                number_of_j += 1
        return number_of_j
            
        

File: Go/easy/771.go
func numJewelsInStones(J string, S string) int {
    var table [130]bool
    for _, j := range[]rune(J) {
        table[j] = true
    }
    ans:= 0
    for _, s := range[]rune(S) {
        if (table[s]) {
            ans++   
        }
    }
    return ans
}